website/docs/r/glue_resource_policy.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Glue"
+layout: "aws"
+page_title: "AWS: aws_glue_resource_policy"
+description: |-
+  Provides a resource to configure the aws glue resource policy.
+---
+
+# Resource: aws_glue_resource_policy
+
+Provides a Glue resource policy. Only one can exist per region.
+
+## Example Usage
+
+### Non-VPC Connection
+
+```hcl
+data "aws_iam_policy_document" "glue-example-policy" {
+  statement {
+    actions = [
+      "glue:CreateTable",
+    ]
+    resources = ["arn:aws:glue:AWS_REGION:ACCOUNT_ID:*"]
```

Comment:
Recommend updating this example to match the updated test configurations with the data sources. 👍